### PR TITLE
Stage 1: User info page

### PR DESCRIPTION
**Pull request by btjl**

### CODE REVIEW

File: src/app/user/(logged-in)/layout.tsx [hunk 1]
```diff
@@ -25,6 +25,12 @@ export default async function Layout(props: { children: React.ReactNode }) {
...
     },
+    {
+      label: "Info",
+      href: RoutePath.USER_INFO,
+      iconSrc: IMG_PATH.SIDE_BAR_INFO,
+      iconLightSrc: IMG_PATH.SIDE_BAR_INFO_LIGHT,
+    },
     {
       label: "Criteria",
       href: RoutePath.USER_CRITERIA,
```
Comment (danui) [inline src/app/user/(logged-in)/layout.tsx:36]: The Criteria and Process options can be removed. They are replaced by Info.

File: src/components/bark/bark-typography.tsx [hunk 2]
```diff
@@ -14,7 +14,7 @@ export function BarkH2(props: { children: React.ReactNode }) {
   );
 }
 
-export function BarkH3(props: { children: React.ReactNode }) {
+export function BarkH3(props: { children: React.ReactNode, }) {
```
Comment (danui) [inline src/components/bark/bark-typography.tsx:17]: Run `make fmt` once. The ensure-pretty check is complaining about this.

https://github.com/barkbank/barkbank-pawtal/actions/runs/8897976730/job/24434099172

File: src/components/bark/bark-typography.tsx [hunk 3]
```diff
@@ -32,12 +32,16 @@ export function BarkH4(props: { children: React.ReactNode }) {
...
+      {props.children}
+    </h5>
   );
 }
 
-export function BarkP(props: { children: React.ReactNode }) {
+export function BarkP(props: { children: React.ReactNode; classes?: string }) {
   return (
-    <p className="leading-7 [&:not(:first-child)]:mt-6">{props.children}</p>
+    <p className={`${props.classes} leading-7 [&:not(:first-child)]:mt-6`}>
```
Comment (danui) [inline src/components/bark/bark-typography.tsx:43]: Why not use `clsx` here?

**Review by danui — COMMENTED** [review]
Just some small things to fix-up. Otherwise, it looks great!

File: src/components/bark/bark-typography.tsx (hunk 2 above)
Comment (btjl) [reply src/components/bark/bark-typography.tsx]: Done! 1584779139fe1ad5a2c9d78a8f07aedd82f3c69d

File: src/components/bark/bark-typography.tsx (hunk 3 above)
Comment (btjl) [reply src/components/bark/bark-typography.tsx]: Forgot about this package! Done 1584779139fe1ad5a2c9d78a8f07aedd82f3c69d

File: src/app/user/(logged-in)/layout.tsx (hunk 1 above)
Comment (danui) [reply src/app/user/(logged-in)/layout.tsx:36]: Bump.

Comment (btjl) [reply src/app/user/(logged-in)/layout.tsx]: Done! 9adb788fd8d1d822aa5fd078179a337658403588

Comment (danui) [reply src/app/user/(logged-in)/layout.tsx]: Process also. (Was not able to highlight it for some reason 🙏)

Thanks!

**Review by danui — APPROVED** [review]

File: src/app/user/(logged-in)/layout.tsx (hunk 1 above)
Comment (btjl) [reply src/app/user/(logged-in)/layout.tsx]: Done! 4fbc07ba293a22f616be32dbe51528cd19f3c574

**Pull request merged**